app/courses/[courseId]/student.tsx

```diff
@@ -358,29 +351,30 @@ function StudentSection({
 
   // Workaround until the backend calculates the progress
```

Comment:
Is it still a workaround? If not, we should remove this comment

Reply:
Immer noch ein Workaround. Nur ein etwas schönerer

Reply:
Ok verstanden